app/src/main/java/com/mapbox/maps/testapp/examples/terrain3D/Terrain3DShowcaseActivity.kt

```diff
@@ -40,6 +41,10 @@ class Terrain3DShowcaseActivity : AppCompatActivity() {
         }
         +atmosphere { }
         +projection(ProjectionName.GLOBE)
+        +light {
+          castShadows(true)
+          shadowIntensity(10.0)
+        }
```

Comment:
not sure we should add shadow to the terrain activity, and seems the terrain doesn't work well with shadows?

Reply:
removed from terrain activity.